lib/charms/mongodb/v0/helpers.py

```diff
@@ -188,6 +188,7 @@ def build_unit_status(mongodb_config: MongoDBConfiguration, unit_ip: str) -> Sta
 
 def copy_licenses_to_unit():
     """Copies licenses packaged in the snap to the charm's licenses directory."""
+    subprocess.check_output("mkdir -p src/licenses", shell=True)
```

Comment:
Any reason not to use the native python method?

Reply:
Just to make code look similar. Will update

Reply:
Done